pkg/config/types.go

```diff
@@ -179,4 +179,10 @@ type Config struct {
 	DynamicFilePath string
 	//use UserId for mapping, IdentityArn is not used any more when DynamicFileUserIDStrict=true
 	DynamicFileUserIDStrict bool
+	//defines if Enforce the username not to start with the reserved prefixes. if the value is true,
+	//then username with the reserved prefixes will get access denied with the error reserved prefix in username
+	DynamicFileUsernamePrefixEnforce bool
+	//defines the reserved prefixes for kubernetes username, it is:
+	//"amazon", "aws", "eks","iam","system"
```

Comment:
This set of strings might be similar to EKS's config, but not a default anywhere in this project. Can you remove this line?